src/main/java/uk/ac/ebi/ena/readtools/sampler/StripesDataEater.java

```diff
@@ -19,16 +19,16 @@
 
 import org.apache.commons.compress.compressors.bzip2.BZip2CompressorOutputStream;
 
-import uk.ac.ebi.ena.readtools.loader.common.eater.DataEater;
-import uk.ac.ebi.ena.readtools.loader.common.eater.DataEaterException;
+import uk.ac.ebi.ena.readtools.loader.common.eater.DataConsumer;
+import uk.ac.ebi.ena.readtools.loader.common.eater.DataConsumerException;
 import uk.ac.ebi.ena.readtools.loader.fastq.DataSpot;
 import uk.ac.ebi.ena.readtools.sampler.bits.BaseDict;
 import uk.ac.ebi.ena.readtools.sampler.bits.BitOutputStream;
 import uk.ac.ebi.ena.readtools.sampler.bits.DictionaryOutputStream;
 import uk.ac.ebi.ena.readtools.sampler.bits.ScoreDict;
 
 public class 
-StripesDataEater<T1 extends DataSpot, T2> implements DataEater<T1, T2>
+StripesDataEater<T1 extends DataSpot, T2> implements DataConsumer<T1, T2>

```

Comment:
Remove StripesDataEater. Used only by StripesDataEater that can also be removed.

Reply:
This is in use by SplitReader.